src/pages/references/stacks-node-configuration.md

```diff
@@ -233,6 +239,26 @@ Example:
 prometheus_bind = "0.0.0.0:9153"
 ```
 
+#### deny_nodes (optional)
+
+List of nodes that must not be added to neighbors
+
+Example:
+
+```toml
+deny_nodes = ???
```

Comment:
Let's change the example to something closer to what the user might want to use.
```suggestion
deny_nodes = 1.2.3.4,exclude.this.node.org
```